src/Middleware/NavigationMiddlewareFactory.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Zend\Navigation\Middleware;
+
+use Interop\Container\ContainerInterface;
+use Zend\Navigation\Navigation;
+
+class NavigationMiddlewareFactory
+{
+    /**
+     * Top-level configuration key indicating navigation configuration
+     *
+     * @var string
```

Comment:
#### phpDocumentor

`@const` is not documented and supported.

#### PHP-Fig

> You may use the @var tag to document the "Type" of the following "Structural Elements":
> - Constants, both class and global scope
> - Properties
> - Variables, both global and local scope

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#722-var